README.md

```diff
@@ -9,259 +16,41 @@
 
 ## Introduction
 
-The OCM provider allows Terraform to manage Red Hat OpenShift Service on AWS (ROSA) clusters, machine pools and identity provider.
+The Red Hat OCM provider allows Terraform to manage Red Hat OpenShift Service on AWS (ROSA) clusters, machine pools, and identity provider.
 
-## OCM
+More information about ROSA, see the Red Hat documentation [here](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-understanding).
 
-Red Hat OpenShift Cluster Manager is a managed service where you can install, modify, operate, and upgrade your Red Hat OpenShift clusters. This service allows you to work with all of your organization’s clusters from a single dashboard. More information can be found [here](https://docs.openshift.com/rosa/ocm/ocm-overview.html).
+## Prerequisites 
+* [GoLang version 1.20 or newer](https://go.dev/doc/install)
+* [Terraform version 1.4.6 or newer](https://developer.hashicorp.com/terraform/downloads)
+* An AWS account
+* ROSA CLI
```

Comment:
There is an article ("an") before AWS account whereas ROSA CLI doesn't have an article (probably "the"?). This may or may not be a big deal.